src/common/input/xkb_mapper.cpp

```diff
@@ -14,7 +14,7 @@
  * along with this program.  If not, see <http://www.gnu.org/licenses/>.
  */
 
-#include "mir/input/xkb_mapper.h"
+#include "mir/input/xkb_mapper_deprecated.h"
```

Comment:
Why would we rename the file and break downstreams that use it?

Reply:
So the tricky part is that the import will be the same when we include `mir/input/xkb_mapper.h`. So one of them has to be named differently, and I chose the old one.

Would it be a better idea to rename the new one to something slightly different? Or is there a better solution?

Reply:
So, I thought "we don't change old stuff, just deprecate it". New stuff can be called by new names, old stuff cannot